hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/client/HoodieFlinkWriteClient.java

```diff
@@ -346,23 +344,21 @@ protected void postCommit(HoodieTable<T, List<HoodieRecord<T>>, List<HoodieKey>,
   @Override
   public void commitCompaction(
       String compactionInstantTime,
-      List<WriteStatus> writeStatuses,
-      Option<Map<String, String>> extraMetadata) throws IOException {
+      HoodieCommitMetadata metadata,
```

Comment:
If we don't care about the generic let's mark it as wildcard instead of using it as raw type: `HoodieCommitMetadata<?>`

Reply:
Sorry, Alexey. I didn't get the point.
Did you mean `HoodieWriteMetadata` which is `public class HoodieWriteMetadata<O> {xxx}`
Or `HoodieCommitMetadata ` which is `public class HoodieCommitMetadata implements Serializable {xxx}` :<

Reply:
I was referring to line 347, you're adding generic class `HoodieCommitMetadata` parameter and using it as a raw-type, instead it should be `HoodieCommitMetadata<?>`

Reply:
Caught up offline: my comment was misattributed